city_scrapers/spiders/ind_school_board.py

```diff
@@ -65,16 +66,19 @@ def parse(self, response):
                 )
 
                 meeting["status"] = self._get_status(meeting)
+                #meeting["status"] = 'passed'
```

Comment:
Looks like dead code! Please make sure to delete commented out lines like these 🙏

Reply:
@SimmonsRitchie Totally good. 
For what it's worth, I kept this specific comment in because I wanted you to see the line that would allow ind_school_board to successfully run, in contrast to line#68 that was making it fail.

Reply:
Ah, I see, @haileyhoyat. Thank you for the explanation. I understand where you were coming from but in future I'd be very appreciative if you try to avoid committing commented out code or print statements or other working/debugging changes. The cleaner our commit history is, the easier it is for reviewers to review changes and for contributors to understand changes over time. It also decreases the possibility of introducing unexpected bugs or dead code in the final PR (it's easy to overlook things like that if you've been staring at code for a long time).

I know it's a little bit more laborious but if you need to explain a bug or unexpected behavior, re: line 68/69, it's probably best to leave a note in the PR notes, make a new comment on the PR, or leave a comment directly on the line in the "files changed" section of the PR. You can always leave code examples in the comment itself (which is often very helpful!)